nss-day-cohort-29/welcome-to-nashville-cyclones
# set up grunt directories

#What:
Set up project's initial file system 

#Why:
Run Grunt 

#Steps To Test:
1. clone repo
1. git fetch -all
1. git checkout attempt1
1. check that there is a README.md and .gitignore file in the root directory
check that there are src and api directories within the root directory
check that there is a lib, scripts and styles directories in the src directory
...
